include/simdjson/generic/ondemand/raw_json_string-inl.h

```diff
@@ -13,26 +13,142 @@ simdjson_really_inline simdjson_warn_unused simdjson_result<std::string_view> ra
   return result;
 }
 
-simdjson_really_inline simdjson_warn_unused simdjson_result<std::string_view> raw_json_string::unescape(json_iterator &iter) const noexcept {
-  return unescape(iter.string_buf_loc());
+simdjson_really_inline bool raw_json_string::is_free_from_unescaped_quote(std::string_view target) noexcept {
```

Comment:
As noted before, I am not sure this is a relevant restriction if the user is given the right constraints. This, on the other hand, might help:

* is_normalized_json_string(key): validates that your string does not contain invalid characters, has no \u escapes (except for some under 0x20 with no escape character equivalent, I guess?), and is not terminated by an odd number of backslashes.